### ISSUE TITLE
my laptop ran into this and i have no idea how to solve

### ISSUE THREAD

**Issue by DatNeedsHelp** [cheat-engine/cheat-engine#2020]
![Screenshot 2022-01-28 183744](https://user-images.githubusercontent.com/98589822/151540549-3f9ff97a-c395-42be-9d01-379cb7e5cc1f.png)

**Comment by CamilleHbp** [comment]
Same here.

**Comment by fade2gray** [comment]
There are two possibilities that I can think of…

1. You are downloading the installer, but disconnecting your internet connection before running it.
2. You have a strict firewall policy that blocks outgoing connections for 'unknown' apps, for which you will need to add an exception for the installer.

**Comment by fade2gray** [comment]
@CamilleHbp No good sending me any logs, I'm afraid I'm just another end-user.

`Runtime error 216 at 0040844A` seems to indicate that your system has a virus, you can search online to confirm it.

I would suggest going [here](https://forums.malwarebytes.com/topic/9573-im-infected-what-do-i-do-now/) and follow the instructions to download and use the `Farbar Recovery Scan tool`, and then follow the rest of the guide on how to report the results – note: you do not need to install Malwarebytes to use the tool. Also, Windows will warn you against downloading and running the tool.

If you do manage to sort this out, it would be nice to hear how things turned out

**Comment by CamilleHbp** [comment]
Okay, I understand why now. Instead of being a regular installer, this thing downloads the actual program over the web. And their content server prevents you from using a VPN. Annoying as hell, but at least my problem is fixed. They should just provide the entire binary though, I don't understand why everyone has those 2MB installers that download over the net.

**Comment by fade2gray** [comment]
Yes, it's an online installer that advertises softwares that the user might be interested in, and though the user can opt out of installing those softwares, the advertising helps generate a little income for the author; if this is too inconvenient, you could always compile your own binaries.

If the above is too much hassle, you could always subscribe to [patreon.com/cheatengine](https://patreon.com/cheatengine) to get the binaries.

**Comment by CamilleHbp** [comment]
Thanks for the information! Didn't know you could get the binaries from patreon, which is less annoying than compiling the program each time :) To be honest, I wasn't annoyed at the advertising, but at the fact I had to switch off my VPN to download the installer :)

Thanks for your kind help!
